RULES.md

```diff
@@ -0,0 +1,153 @@
+Everything in the [based.cooking](https://based.cooking) repository is licensed under the CC0 1.0 license.
+Contributions licensed under any incompatible license will be rejected.
+
+#### Requests for removal
+
+Requests to have your work removed will be ignored.
+
+#### Attribution
+
+Attribution is unnecessary under the CC0 1.0 license.
+However, you are welcome and recommended to give yourself credit at the bottom of a recipe you submitted, including links to your personal website and to your donation pages.
+**Do not** link to social media sites.
+
+### Recipe format
+
+Model submission files after [example.md](example.md), then put them in `src/`.
+
+### Markdown style
+
+Recipes should start with a title, with a single `#` *on the first line*.
+
+### Technical details
+
+#### File format
+
+- Recipes should start with no empty line at the top and end with no trailing lines.
+- There should be no trailing whitespace.
+- The file needs to be `\n` terminated in Unix-like fashion (if you're using a Unix-like system, it should be automatic).
+- Encode files in UTF-8.
+
+#### File name
+
+- File names should be the name of the dish with words separated by hyphens (`-`), not underscores, and definitely not spaces.
+
+#### Tags
+
+- Recipes should end with a list of relevant tags.
+The syntax is:
+
+```md
+;tags: tag1 tag2 tag3
+```
+
+- The tag line should be a single line, at the end of the markdown file, preceded by a blank line.
+- Add between a few (1-4) tags.
+- **Prioritize existing tags**.
+- As a general guideline, add the country from which the recipe originates if the recipe is representative of said country, using the adjective form (eg. *mexican*, *italian*, etc).
+- Tag the main ingredient if it's something slightly special.
+
+##### List of special, categorical tags to use if relevant
+
+- `basic`: for basic recipes that aren't meant to be standalone but are supposed
+  to be incorporated in another recipe.
+- `breakfast`
+- `cheesefare`: for recipes that include dairy but no milk.
+- `dessert`
+- `drink`
+- `quick`: for recipes that can be cooked in under ~20 minutes.
+- `side`: side dishes such as mash, fries, etc.
+- `snack`
+- `spread`
+- `strict-fast`: for recipes that contain no meat or dairy.
+- `vegan`
+- `vegetarian`
+
+#### Images
+
+- Images are stored in `data/pix`.
+- Each recipe can have a title image at the top and perhaps several instructional images as absolutely necessary.
+- Do not add stock images you found on the internet.
+Take a good picture yourself of the actual dish you prepared.
+- If you see a bad or substandard image, you may submit a better one.
+- Images should be in `.webp` format and with as small a file size as possible.
+If you submit an image for say, `chicken-parmesan.md`, it should be added as `pix/chicken-parmesan.webp`.
+- 150Kb is the maximum file size for a single image.
```

Comment:
```suggestion
- 150kB is the maximum file size for a single image.
```

technically lowercase `b` is `bits`, so just want to avoid possible confusion. I think the `k` is usually lowercase too, but w/e. There's only 1 k prefix so doesn't matter that much.

Reply:
AFAIK this is the official prefix for metric kilobytes.